src/main/java/org/example/postory/domain/comment/service/CommentServiceImpl.java

```diff
@@ -108,4 +106,27 @@ public void likeComment(long commentId, UserDetails userDetails) {
             commentLikeRepository.save(new CommentLike(user, comment));
         }
     }
+
+    /**
+     * [Service] 댓글 수정 메서드. 주어진 댓글 ID에 해당하는 댓글을 찾아, 작성자가 맞는 경우에만 내용을 수정
+     *
+     * @param authUserId 댓글을 수정하려는 사용자 ID
+     * @param requestDto 수정할 댓글 내용을 담은 DTO
+     * @param commentId  수정 대상 댓글의 ID
+     * @return 수정된 댓글 정보를 담은 CommentItem DTO
+     * @throws ApiException {@code ErrorType.FORBIDDEN_COMMENT} - 요청한 사용자가 댓글의 작성자가 아닌 경우
+     */
+    @Override
+    public CommentItem updateComment(Long authUserId,
+        CommentRequestDto.CommentItem requestDto, Long commentId) {
+        Comment comment = commentRepository.getCommentByIdOrElseThrow(commentId);
+
+        //작성자 본인만 수정가능
+        if (!authUserId.equals(comment.getUser().getId())) {
+            throw new ApiException(ErrorType.FORBIDDEN_COMMENT);
+        }
+
+        comment.setContent(requestDto.getContents());
+        return new CommentItem(commentRepository.save(comment));
```

Comment:
@Transaction 붙여주면 update 했을 때 변경 감지 통해서 자동으로 DB에 반영해줘서 save() 안해줘도 될 것 같아요! 기존 comment 객체도 새로운 값으로 바뀐다고 하네요..!
```suggestion
        return new CommentItem(comment);
```